Changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 7.6.0
+
+- Add `instructable_embed` to `Client` and `AsyncClient`
```

Comment:
Would be great if you could add a bit more context, e.g. what this endpoint is used for. In the end this changelog is user facing.

Reply:
I just followed the existing changelog format which is rather brief. Details about the new function including an example can be found in the doc strings.